charts/airbyte/values.yaml

```diff
@@ -7,11 +7,11 @@ global:
   serviceAccountName: &service-account-name "airbyte-admin"
   deploymentMode: "oss"
 
-  secretName: "airbyte-secrets"
+  secretName: ""
```

Comment:
what's happening to these values?

Reply:
I'll work on the documentation in my next PR, I just left it blank, so the default value for secrets will be used